tools/sense_studio/project_utils.py

```diff
@@ -31,11 +31,89 @@ def lookup_project_path(project_name):
     return projects[project_name]['path']
 
 
+def _backwards_compatibility_update(path, config):
+    updated = False
+
+    if 'use_gpu' not in config:
+        config['use_gpu'] = False
+        updated = True
+
+    if 'temporal' not in config:
+        config['temporal'] = False
+        updated = True
+
+    if 'assisted_tagging' not in config:
+        config['assisted_tagging'] = False
+        updated = True
+
+    if 'video_recording' not in config:
+        config['video_recording'] = {
+            'countdown': 3,
+            'recording': 5,
+        }
+        updated = True
+
+    if 'project_tags' not in config:
+        # Collect class-wise tags
+        old_classes = config['classes']
+        project_tags_list = []
+        for class_name, class_tags in old_classes.items():
+            project_tags_list.extend(class_tags)
+
+        # Assign project-wide unique indices to tags (0 is reserved for 'background')
+        project_tags = {idx + 1: tag_name for idx, tag_name in enumerate(project_tags_list)}
```

Comment:
Maybe `sort` class labels first and then assign them index? What do you say? @corneliusboehm

Reply:
Sure, why not